tasks/components/RoomList.tsx

```diff
@@ -32,7 +40,10 @@ const defaultRoomListTranslations: Record<Languages, RoomListTranslation> = {
     selectAll: 'Select All',
     rooms: 'Rooms',
     addRoom: 'Add Room',
-    bedCount: 'Number of Beds'
+    bedCount: 'Number of Beds',
+    dangerZoneText: 'Deleting room(s) is a permanent action and cannot be undone. Be careful!',
+    deleteConfirmText: 'Do you really want to delete the selected room(s)?',
+    deleteRoom: 'Delete Room(s)',
```

Comment:
We should not use pluralize via an `(s)`. Most of the time, we already have the context of how many entities this action will affect. In that case, there is also already the singularized `room`.